<launch>
 <node name="uvc_camera_node" pkg="uvc_camera"  	 	type="uvc_camera_node" output="screen">
   	 <param name="camera_info_url" value="file://$(find 	   beginner_tutorials)/config/camera.yaml"/>
 </node>

  <node name="image_proc" pkg="image_proc" type="image_proc" output="screen"/>
  <node name="image_rect_view" pkg="image_view" type="image_view" >
    <remap from="image" to="image_rect_color"/>
  </node>

  <node pkg="lower_camera" name="line_trace" type="line_trace.py">
  </node>

  <node name="image_view" pkg="image_view" type="image_view" output="screen" >
  <remap from="image" to="line_image"/>
  </node>

  <node pkg="thruster" name="thruster_control" type="thruster_control.cpp">
  </node>
</launch>
